static/app/components/events/interfaces/frame/assembly.tsx

```diff
@@ -35,12 +35,14 @@ function Assembly({name, version, culture, publicKeyToken, filePath}: Props) {
       </AssemblyInfo>
 
       {filePath && (
-        <FilePathInfo>
+        <AssemblyInfo>
           <Caption>{t('Path')}:</Caption>
           <Tooltip title={filePath}>
-            <TextCopyInput rtl>{filePath}</TextCopyInput>
+            <TextCopyInput rtl size="xs">
+              {filePath}
+            </TextCopyInput>
           </Tooltip>
-        </FilePathInfo>
+        </AssemblyInfo>
```

Comment:
Tweaking this because the `TextCopyInput` update breaks some padding rules in `FilePathInfo`.

**Before:**
<img width="560" alt="Screen Shot 2022-08-19 at 2 57 02 PM" src="https://user-images.githubusercontent.com/44172267/185712562-596bb685-b9b4-44a5-bd73-4450c2728d20.png">

**After:**
<img width="560" alt="Screen Shot 2022-08-19 at 2 55 45 PM" src="https://user-images.githubusercontent.com/44172267/185712592-0f59a0fb-a1e4-4cc5-b61d-86b2cbeddfb3.png">